mambu/api.py

```diff
@@ -1,21 +1,852 @@
+            the client to create in mambu
+        addresses: list(ClientAddress)
+            list of length 1 containing an
+        custom_information: list(CustomField)
+            list of custom fields to set for the client
+        id_documents: list(IdDocument)
+            list of id documents to attach to the customer account
+
+        Returns
+        -------
+        dict
+        """
+        return self._create_or_update_client(
+            None, client, addresses, custom_information, id_documents)
+
+    def update_client(self, client_id, client, addresses=None,
+                      custom_information=None, id_documents=None):
+        """Update the client in mambu with client_id using the data provided in
+        the parameters
+        WARNING This is a post action that will overwrite and existing data for
+        teh client
+
+        Parameters
+        ----------
+        client_id: int, str
+            id or encoded_key of client in mambu
+        client: Client
+            object contain the standard Client fields
+        addresses: list(ClientAddress)
+            list of length one containing ClientAddress data
+        custom_information: list(CustomField)
+            list of CustomField objects to overwrite in mambu
+        id_documents: list(IdDocument)
+            list of documents to attach to the customer account in mambu
+
+        Returns
+        -------
+        dict
+        """
+        return self._create_or_update_client(
+            client_id, client, addresses, custom_information, id_documents)
+
+    def _create_or_update_client(
+            self, client_id=None, client=None, addresses=None,
+            custom_information=None, id_documents=None):
+        """Either create or update a client in mambu
+
+        Parameters
+        ----------
+        client_id: None, int, str
+            id or encoded_key of client in mambu
+        client: Client
+            object contain the standard Client fields
+        addresses: list(ClientAddress)
+            list of length one containing ClientAddress data
+        custom_information: list(CustomField)
+            list of CustomField objects to overwrite in mambu
+        id_documents: list(IdDocument)
+            list of documents to attach to the customer account in mambu
+
+        Returns
+        -------
+        dict
+        """
+        return self._post(self._url_clients(client_id), data=dict(
+                client=client, idDocuments=id_documents, addresses=addresses,
+                customInformation=custom_information))
+
+    def set_client_custom_field(self, client_id, custom_field_id, value,
+                                index=None):
+        """Sets the value of the customFieldId specified.  If the customFieldId
+        is within a fieldset then index must be set to customFieldSetGroupIndex
+
+        In order to create a new customfield associated with a field set index
+        must be set to -1
```

Comment:
Shall we leave this comment as is? Or remove it in the knowledge that they're removing this 'feature'? Not sure myself.